app/page.tsx

```diff
@@ -62,13 +62,14 @@ export default function HomePage() {
   const router = useRouter();
   const showClientUI = useShowClientUI();
   const [isPopoverOpen, setPopoverOpen] = useState(false);
-  const [category, setCategory] = useState<CategoryOptions>("");
 
   const searchParams =
     typeof window !== "undefined"
       ? new URLSearchParams(window.location.search)
       : new URLSearchParams("/");
 
+  const [category, setCategory] = useState(searchParams.get("c") || "");
```

Comment:
Is there a reason for us to use `c` instead `category` as search param?

Reply:
we are using only one letter on search params.

No big reason, I just found it less verbose, and for URLs usually I like them short. 

The trade off is you lose a bit of readability but I don't expect people to create the url themselves, and for the super curious once you learn it, it's easier to write.